jmh.gradle

```diff
@@ -17,7 +17,14 @@
  * under the License.
  */
 
-def jmhProjects = [ project("iceberg-spark2") ]
+def jmhProjects = []
+if (JavaVersion.current() == JavaVersion.VERSION_1_8) {
+  jmhProjects = [ project("iceberg-spark2") ]
```

Comment:
I _think_ Spark 3 supports JDK 1.8, would it make sense to do iceberg-spark3 here as well? Or is there a reason for only doing one of them?

Reply:
It would indeed make sense.

I was having trouble getting both projects to work at one time, so I chose to just skip it as that's about as far as my gradle abilities got me. I tried adding to the array, etc, but the plugin didn't like that.

I'll look into it further to see if it can be supported. Thanks for the review!

Reply:
Looking into it again, I think probably the issue was that I didn't run `./gradlew clean` when switching between JDKs locally. Pushing a fix now!

Reply:
Ah. Confirmed the issue is 100% with my own understanding of groovy / gradle (and always wanting it to be more like Scala).

Reply:
Alright. This has been fixed :). Confirmed by setting my terminals JDK to java 8, then running both a spark 2 and spark 3 JMH test, then setting to Java 11, and running a Spark 3 test (and back again). 👍